dist/ci/travis_script.sh

```diff
@@ -46,15 +39,16 @@ if test -z "$SUBTEST"; then
       pushd ../backend
       make test_unit
       ;;
+    migrate)
+      mv db/structure.sql db/structure.sql.git
+      bundle exec rails db:drop db:create || exit 1
+      xzcat test/dump_2.5.sql.xz | sed '/^USE api_.*;/d' | mysql ci_api_test
```

Comment:
@adrianschroeter didn't you recently have a PR which updated migration 001 so that it matched the 2.5 database? What exactly is different between the 2.5 dump and 001 migration?